spotinst/ocean_gke_launch_spec/fields_spotinst_ocean_gke_launch_spec.go

```diff
@@ -1132,6 +1132,92 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 		},
 		nil, nil, nil, nil,
 	)
+
+	fieldsMap[NetworkInterfaces] = commons.NewGenericField(
+		commons.OceanGKELaunchSpec,
+		NetworkInterfaces,
+		&schema.Schema{
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					string(Network): {
+						Type:     schema.TypeString,
+						Required: true,
+					},
+
+					string(ProjectId): {
+						Type:     schema.TypeString,
+						Optional: true,
+					},
+
+					string(LaunchSpecAccessConfigs): {
+						Type:     schema.TypeSet,
+						Optional: true,
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								string(LaunchSpecAccessConfigsName): {
+									Type:     schema.TypeString,
+									Optional: true,
+								},
+
+								string(Type): {
+									Type:     schema.TypeString,
+									Optional: true,
+								},
+							},
+						},
+					},
+
+					string(LaunchSpecAliasIPRanges): {
+						Type:     schema.TypeSet,
+						Optional: true,
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								string(IPCidrRange): {
+									Type:     schema.TypeString,
+									Required: true,
+								},
+
+								string(SubnetworkRangeName): {
+									Type:     schema.TypeString,
+									Required: true,
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		func(resourceObject interface{}, resourceData *schema.ResourceData, meta interface{}) error {
```

Comment:
refer any other similar block of code for read

Reply:
added